lib/bundle/brewfile.rb

```diff
@@ -2,29 +2,39 @@
 
 module Bundle
   module Brewfile
+    BUNDLE_FILE_ENV_KEY = "HOMEBREW_BUNDLE_FILE"
```

Comment:
Again, please hardcode this: https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)

Reply:
as you like